cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -431,6 +440,14 @@ public UUID userTaskId() {
       return _queryParams;
     }
 
+    private EndPoint endPoint() {
+      return _endPoint;
+    }
+
+    private long executionTime() {
```

Comment:
I propose 2 changes: (1) public getter (2) `Ns` at the end:
`private long executionTime() {` -> `public long executionTimeNs() {`